internal/venice-common/src/main/java/com/linkedin/venice/writer/VeniceWriter.java

```diff
@@ -304,7 +304,11 @@ public VeniceWriter(
     this.isChunkingEnabled = params.isChunkingEnabled();
     this.isChunkingSet = true;
     this.isRmdChunkingEnabled = params.isRmdChunkingEnabled();
-    // this.maxRecordSizeBytes = props.getInt(MAX_RECORD_SIZE_BYTES, DEFAULT_MAX_RECORD_SIZE_BYTES);
+    int maxRecordSizeBytesFromProps = props.getInt(MAX_RECORD_SIZE_BYTES, DEFAULT_MAX_RECORD_SIZE_BYTES);
+    if (maxRecordSizeBytesFromProps == -1) {
+      maxRecordSizeBytesFromProps = DEFAULT_MAX_RECORD_SIZE_BYTES;
```

Comment:
Is this logic still required after Controller backfills the default value in `StoreResponse`?

Reply:
Removed. Missed `VeniceWriter`.